meta-refkit/conf/bblayers.conf.sample

```diff
@@ -15,6 +15,7 @@ REFKIT_LAYERS = " \
   ##OEROOT##/../meta-iotqa \
   ##OEROOT##/../meta-security-isafw \
   ##OEROOT##/../meta-intel-realsense \
+  ##OEROOT##/../meta-clang \
```

Comment:
Hmm, do you mean that we need to bump up `LCONF_VERSION` to 2 in `bblayers.conf.sample`?